packages/expo-doctor/src/utils/doctorConfig.ts

```diff
@@ -17,13 +17,21 @@ import { gteSdkVersion } from './versions';
  *        "enabled": true,
  *        "exclude": ["/foo/", "bar"]
  *        "listUnknownPackages": true,
- *      }
+ *      },
+ *      "cngCheckEnabled": false
  *    }
  *  }
  * ```
  **/
 
-export function getDoctorConfig(pkg: any) {
+export function getDoctorConfig(pkg: any): {
+  reactNativeDirectoryCheck?: {
+    enabled?: boolean;
+    exclude?: string[];
+    listUnknownPackages?: boolean;
+  };
+  cngCheckEnabled?: boolean;
```

Comment:
Thinking more that we should name the config closer to the name of the check in this case, and in the same format as `reactNativeDirectoryCheck`, like:
`appConfigFieldsNotSyncedCheck: { enabled?: boolean }`

(since this check is a very specific check about how you're CNG'ing, and not exhaustively covering all things CNG)